### PR TITLE
Global surface mooring MET-CT updates

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: GI01SUMO/GI01SUMO_R00003_ingest.csv [hunk 1]
```diff
@@ -78,5 +80,7 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SU
 mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00003/cg_data/dcl12/pco2a/*.pco2a.log,GI01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00003/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00003/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00003/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00003/instruments/dcl12/METBK-CT*/*.hex,GI01SUMO-SBD12-06-METBKA001,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_R00003_ingest.csv:84]: looks good

File: GI01SUMO/GI01SUMO_R00004_ingest.csv [hunk 2]
```diff
@@ -78,5 +80,7 @@ mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/O
 #mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00004/cg_data/dcl12/pco2a/*.pco2a.log,GI01SUMO-SBD12-04-PCO2AA000,recovered_host,Not Expected,buoy lost: not recovered
 #mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00004/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Not Expected,buoy lost: not recovered
 #mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00004/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Not Expected,buoy lost: not recovered
+#mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00004/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA001,recovered_host,Not Expected,buoy lost: not recovered
+#mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00004/instruments/dcl12/METBK-CT*/*.hex,GI01SUMO-SBD12-06-METBKA001,recovered_inst,Not Expected,buoy lost: not recovered
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_R00004_ingest.csv:84]: looks good

File: GI01SUMO/GI01SUMO_R00005_ingest.csv [hunk 3]
```diff
@@ -78,5 +80,7 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SU
 mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/cg_data/dcl12/pco2a/*.pco2a.log,GI01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/instruments/dcl12/METBK-CT*/*.hex,GI01SUMO-SBD12-06-METBKA001,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_R00005_ingest.csv:84]: looks good

File: GI01SUMO/GI01SUMO_R00006_ingest.csv [hunk 4]
```diff
@@ -78,5 +80,7 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SU
 mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/pco2a/*.pco2a.log,GI01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/instruments/dcl12/METBK-CT*/*.hex,GI01SUMO-SBD12-06-METBKA001,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_R00006_ingest.csv:84]: looks good

File: GI01SUMO/GI01SUMO_R00007_ingest.csv [hunk 5]
```diff
@@ -78,5 +80,7 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SU
 mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl12/pco2a/*.pco2a.log,GI01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl12/METBK-CT*/*.hex,GI01SUMO-SBD12-06-METBKA001,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_R00007_ingest.csv:84]: looks good

File: GI01SUMO/GI01SUMO_R00008_ingest.csv [hunk 6]
```diff
@@ -78,5 +80,7 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SU
 mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/cg_data/dcl12/pco2a/*.pco2a.log,GI01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/instruments/dcl12/METBK-CT*/*.hex,GI01SUMO-SBD12-06-METBKA001,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_R00008_ingest.csv:84]: looks good

File: GS01SUMO/GS01SUMO_D00001_ingest.csv [hunk 7]
```diff
@@ -50,4 +51,5 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01
 mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00001/cg_data/dcl12/pco2a/*.pco2a.log,GS01SUMO-SBD12-04-PCO2AA000,telemetered,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00001/cg_data/dcl12/wavss/*.wavss.log,GS01SUMO-SBD12-05-WAVSSA000,telemetered,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00001/cg_data/dcl12/metbk2/*.metbk*.log,GS01SUMO-SBD12-06-METBKA000,telemetered,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GS01SUMO/D00001/cg_data/dcl12/metbk2/*.metbk*.log,GS01SUMO-SBD12-06-METBKA001,telemetered,Available,
```
Comment (rtravis-whoi) [inline GS01SUMO/GS01SUMO_D00001_ingest.csv:54]: looks good

File: GS01SUMO/GS01SUMO_D00002_ingest.csv [hunk 8]
```diff
@@ -48,4 +49,5 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01
 mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00002/cg_data/dcl12/pco2a/*.pco2a.log,GS01SUMO-SBD12-04-PCO2AA000,telemetered,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00002/cg_data/dcl12/wavss/*.wavss.log,GS01SUMO-SBD12-05-WAVSSA000,telemetered,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00002/cg_data/dcl12/metbk2/*.metbk*.log,GS01SUMO-SBD12-06-METBKA000,telemetered,Available,
-mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00002/cg_data/dcl12/fdchp/*.fdchp.log,GS01SUMO-SBD12-08-FDCHPA000,telemetered,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GS01SUMO/D00002/cg_data/dcl12/metbk2/*.metbk*.log,GS01SUMO-SBD12-06-METBKA001,telemetered,Available,
```
Comment (rtravis-whoi) [inline GS01SUMO/GS01SUMO_D00002_ingest.csv:52]: looks good

File: GS01SUMO/GS01SUMO_D00003_ingest.csv [hunk 9]
```diff
@@ -48,4 +49,5 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01
 mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00003/cg_data/dcl12/pco2a/*.pco2a.log,GS01SUMO-SBD12-04-PCO2AA000,telemetered,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00003/cg_data/dcl12/wavss/*.wavss.log,GS01SUMO-SBD12-05-WAVSSA000,telemetered,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00003/cg_data/dcl12/metbk2/*.metbk2.log,GS01SUMO-SBD12-06-METBKA000,telemetered,Available,
-mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00003/cg_data/dcl12/fdchp/*.fdchp.log,GS01SUMO-SBD12-08-FDCHPA000,telemetered,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GS01SUMO/D00003/cg_data/dcl12/metbk2/*.metbk2.log,GS01SUMO-SBD12-06-METBKA001,telemetered,Available,
```
Comment (rtravis-whoi) [inline GS01SUMO/GS01SUMO_D00003_ingest.csv:52]: looks good

File: GS01SUMO/GS01SUMO_D00004_ingest.csv [hunk 10]
```diff
@@ -49,4 +50,5 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01
 mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00004/cg_data/dcl12/pco2a1/*.pco2a1.log,GS01SUMO-SBD12-04-PCO2AA000,telemetered,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00004/cg_data/dcl12/wavss/*.wavss.log,GS01SUMO-SBD12-05-WAVSSA000,telemetered,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00004/cg_data/dcl12/metbk2/*.metbk2.log,GS01SUMO-SBD12-06-METBKA000,telemetered,Available,
-mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GS01SUMO/D00004/cg_data/dcl12/fdchp/*.fdchp.log,GS01SUMO-SBD12-08-FDCHPA000,telemetered,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GS01SUMO/D00004/cg_data/dcl12/metbk2/*.metbk2.log,GS01SUMO-SBD12-06-METBKA001,telemetered,Available,
```
Comment (rtravis-whoi) [inline GS01SUMO/GS01SUMO_D00004_ingest.csv:53]: looks good

File: GS01SUMO/GS01SUMO_R00001_ingest.csv [hunk 11]
```diff
@@ -73,6 +73,8 @@ mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC
 mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/DOSTA_sn379/*.dosta1.log,GS01SUMO-SBD11-04-DOSTAD000,recovered_host,Available,
 mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/SPKIR_sn269/*.spkir1.log,GS01SUMO-SBD11-05-SPKIRB000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/METBK-Log_snLGR018/*.metbk*.log,GS01SUMO-SBD11-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/METBK-Log_snLGR018/*.metbk*.log,GS01SUMO-SBD11-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00001/cg_data/dcl11/METBK-CT*/*.hex,GS01SUMO-SBD11-06-METBKA001,recovered_inst,Not Available,Data not available
```
Comment (rtravis-whoi) [inline GS01SUMO/GS01SUMO_R00001_ingest.csv:77]: line 77 - recovered_instr line should be commented out

File: GS01SUMO/GS01SUMO_R00002_ingest.csv [hunk 12]
```diff
@@ -78,5 +80,7 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SU
 mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00002/cg_data/dcl12/pco2a/*.pco2a.log,GS01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00002/cg_data/dcl12/wavss/*.wavss.log,GS01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00002/cg_data/dcl12/metbk2/*.metbk2.log,GS01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GS01SUMO/R00002/cg_data/dcl12/metbk2/*.metbk2.log,GS01SUMO-SBD12-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00002/instruments/dcl12/METBK-CT*/*.hex,GS01SUMO-SBD12-06-METBKA001,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GS01SUMO/GS01SUMO_R00002_ingest.csv:84]: looks good

File: GS01SUMO/GS01SUMO_R00003_ingest.csv [hunk 13]
```diff
@@ -78,5 +80,7 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SU
 mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl12/pco2a/*.pco2a.log,GS01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl12/wavss/*.wavss.log,GS01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl12/metbk2/*.metbk2.log,GS01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl12/metbk2/*.metbk2.log,GS01SUMO-SBD12-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl12/METBK-CT*/*.hex,GS01SUMO-SBD12-06-METBKA001,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GS01SUMO/GS01SUMO_R00003_ingest.csv:84]: looks good

File: GS01SUMO/GS01SUMO_R00004_ingest.csv [hunk 14]
```diff
@@ -80,5 +82,7 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SU
 mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl12/pco2a1/*.pco2a1.log,GS01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
 mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl12/wavss/*.wavss.log,GS01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl12/metbk2/*.metbk2.log,GS01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl12/metbk2/*.metbk2.log,GS01SUMO-SBD12-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl12/METBK-CT*/*.hex,GS01SUMO-SBD12-06-METBKA001,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GS01SUMO/GS01SUMO_R00004_ingest.csv:86]: looks good

**Review by rtravis-whoi — CHANGES_REQUESTED** [review]
Very minor fixes:
GA01SUMO_R00001
line 77 & line 88 - should this be *.hex instead of *hex?

GA01SUMO_R00002
need to add 'Available' at end of line 84

GS01SUMO_R00001
line 77 - recovered_instr line should be commented out

**Review by rtravis-whoi — APPROVED** [review]
updates look good!

File: GI01SUMO/GI01SUMO_R00005_ingest.csv (hunk 3 above)
Comment (cooleyky) [inline GI01SUMO/GI01SUMO_R00005_ingest.csv:84]: looks good

File: GI01SUMO/GI01SUMO_R00006_ingest.csv (hunk 4 above)
Comment (cooleyky) [inline GI01SUMO/GI01SUMO_R00006_ingest.csv:84]: looks good

File: GI01SUMO/GI01SUMO_R00007_ingest.csv (hunk 5 above)
Comment (cooleyky) [inline GI01SUMO/GI01SUMO_R00007_ingest.csv:84]: looks good

File: GI01SUMO/GI01SUMO_R00008_ingest.csv (hunk 6 above)
Comment (cooleyky) [inline GI01SUMO/GI01SUMO_R00008_ingest.csv:84]: looks good

**Review by cooleyky — COMMENTED** [review]
Looks good except for one issue with GI01SUMO/GI01SUMO_R00002_ingest.csv. Asking @cdobs to check this

File: GI01SUMO/GI01SUMO_R00002_ingest.csv [hunk 15]
```diff
@@ -70,7 +70,7 @@ mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/w
 mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl11/SPKIR_sn288/*.spkir1.log,GI01SUMO-SBD11-05-SPKIRB000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl11/METBK-Log_snLGR021/*.metbk1.log,GI01SUMO-SBD11-06-METBKA000,recovered_host,Available,
 mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl11/METBK-Log_snLGR021/*.metbk1.log,GI01SUMO-SBD11-06-METBKA001,recovered_host,Available,
-mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl11/METBK-CT*/*.hex,GI01SUMO-SBD11-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl11/METBK-CT*/*.hex,GI01SUMO-SBD11-06-METBKA001,recovered_inst,Available,
 mi.dataset.driver.nutnr_b.dcl_full.nutnr_b_dcl_full_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl11/NUTNR_sn274/*.nutnr1.log,GI01SUMO-SBD11-08-NUTNRB000,recovered_host,Available,
```
Comment (cooleyky) [inline GI01SUMO/GI01SUMO_R00002_ingest.csv:74]: update looks good

**Review by cooleyky — APPROVED** [review]
Everything looks good after update to GI01SUMO R00002

File: GI01SUMO/GI01SUMO_R00002_ingest.csv (hunk 15 above)
Comment (rtravis-whoi) [reply GI01SUMO/GI01SUMO_R00002_ingest.csv:74]: looks good!

**Review by rtravis-whoi — APPROVED** [review]
correction to address Kylene's note looks good!

**Pull request merged**